src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -240,6 +244,23 @@ public void onResume() {
 		displayFavouriteTrips();
 		refreshAutocomplete(false);
 
+		// check if there's a preference for selected products for the current network
+		TransportNetwork current_network = Preferences.getTransportNetwork(getContext());
+		String network_pref = current_network.getId().toString() + "_selected_products";
+		SharedPreferences sp = getContext().getSharedPreferences(Preferences.NETWORK_SETTINGS, 0);
```

Comment:
Ok thanks in this case, I prefer `MODE_PRIVATE` because the meaning is clearer when just looking at it than just `0`.